.github/workflows/continuous-integration.yml

```diff
@@ -2,6 +2,18 @@ name: Continuous Integration Pipeline
 
 on:
   push:
+    paths-ignore:
+      - '**.md'
+      - '**.txt'
+      - '**.json'
+      - '**.yml'
```

Comment:
Maybe we should remove this as changes to the workflow must trigger the pipeline

Reply:
good point - will do